development/ts-migration-dashboard/scripts/build-app.ts

```diff
@@ -159,26 +153,22 @@ async function rebuild({
 
   await fs.emptyDir(FINAL_BUILD_DIRECTORY_PATH);
 
-  try {
```

Comment:
We don't need the try/catch here, since if `rebuild` throws an error, it will already be handled by the `rebuild().catch((error: unknown) => { console.error(error); })` on line 195 below.